core/src/test/java/com/scottlogic/datahelix/generator/core/builders/ConstraintChainBuilder.java

```diff
@@ -16,7 +16,6 @@
 
 package com.scottlogic.datahelix.generator.core.builders;
 
-import com.scottlogic.datahelix.generator.common.SetUtils;
```

Comment:
I presume this change was intentional

Reply:
Yes, IntelliJ told me it was unused and looks like it was correct.